test/unittests/evm_eof_function_test.cpp

```diff
@@ -57,3 +57,70 @@ TEST_P(evm, eof_function_example2)
     EXPECT_GAS_USED(EVMC_SUCCESS, 44544);
     EXPECT_EQ(output, "0000000000000000000000000000000000000000000000000000000000000262"_hex);
 }
+
+TEST_P(evm, callf_stack_size_1024)
+{
+    // CALLF is not implemented in Advanced.
+    if (is_advanced())
+        return;
+
+    rev = EVMC_CANCUN;
+    const auto code = bytecode{"ef0001 010008 020002 0BFF 0004 030000 00 000003FF 00000001"_hex} +
```

Comment:
We should later extend the `eof_header` and `eof_bytecode` to support multiple code sections.

Reply:
Issue added https://github.com/ethereum/evmone/issues/605